cou/apps/app.py

```diff
@@ -338,6 +341,44 @@ async def _check_upgrade(self, target: OpenStackRelease) -> None:
             )
             raise ApplicationError()
 
+    async def _run_package_upgrade_command(self, unit: str, command: str) -> None:
```

Comment:
`_run_package_upgrade_command` and `_upgrade_packages` should not be in this class. Because this class is `OpenStackApplication` and these functions does not apply just for `OpenStackApplication`. I think we should just move them out of class.

Reply:
Moved the function to the newly created `app_utils.py`